facilitator/src/bin/facilitator.rs

```diff
@@ -1001,6 +1008,15 @@ fn intake_transport_from_args(matches: &ArgMatches) -> Result<VerifiableAndDecry
 }
 
 fn transport_for_path(path: StoragePath, identity: Identity) -> Result<Box<dyn Transport>> {
+    // It's quite awkward to omit a key from a map on the Terraform side, so we
```

Comment:
What I mean is not including a key in a map at all, with any value. Instead of having to introduce sentinel values like `default` or the empty string, I'd much rather just not include the `*_IDENTITY` parameter in a config map, if it's not needed. I saw you were using the empty string already in `workflow-manager`, so we could standardize on that.